tracing-opentelemetry/src/main/java/io/micronaut/tracing/opentelemetry/instrument/util/ThreadTracingInvocationInstrumenterFactory.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2017-2022 original authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micronaut.tracing.opentelemetry.instrument.util;
+
+import io.micronaut.context.annotation.Requires;
+import io.micronaut.core.annotation.Internal;
+import io.micronaut.scheduling.instrument.InvocationInstrumenter;
+import io.micronaut.scheduling.instrument.InvocationInstrumenterFactory;
+import jakarta.inject.Singleton;
+
+import static io.micronaut.core.util.StringUtils.FALSE;
+import static io.micronaut.core.util.StringUtils.TRUE;
+
+/**
+ * Enables threads tracing invocation instrumentation.
+ */
+@Requires(beans = TracingInvocationInstrumenterFactory.class)
+@Requires(property = ThreadTracingInvocationInstrumenterFactory.PROPERTY_INSTRUMENT_THREADS, value = TRUE, defaultValue = FALSE)
+@Singleton
+@Internal
+final class ThreadTracingInvocationInstrumenterFactory implements InvocationInstrumenterFactory {
```

Comment:
Duplicate: https://github.com/micronaut-projects/micronaut-tracing/blob/master/tracing-core/src/main/java/io/micronaut/tracing/instrument/util/ThreadTracingInvocationInstrumenterFactory.java

Please delete it from OTEL module.